parlai/tasks/dealnodeal/agents.py

```diff
@@ -46,7 +46,7 @@ class NegotiationTeacher(Teacher):
     def __init__(self, opt, shared=None):
         super().__init__(opt, shared)
         self.datatype = opt['datatype'].split(':')[0]
```

Comment:
in retrospect, i don't think this is the patch. pre-split should be the right choice here...